Don't include optimization flags here; MacPorts will supply its own.

Add forgotten flag to link with libiconv.

Do not build language bindings with swig.
--- Makefile.orig	2015-02-12 07:14:33.000000000 -0600
+++ Makefile	2020-05-04 11:54:34.000000000 -0500
@@ -21,8 +21,8 @@
 include Makefile.cfg
 
 export CC = gcc -pthread
-export CFLAGS += -I. -Wall -Os -ggdb -D_GNU_SOURCE -fPIC
-export LDLIBS += -lncursesw
+export CFLAGS += -I. -Wall -ggdb -D_GNU_SOURCE -fPIC
+export LDLIBS += -lncursesw -liconv
 
 SONAME  := libstfl.so.0
 VERSION := 0.24
@@ -73,18 +73,6 @@
 include spl/Makefile.snippet
 endif
 
-ifeq ($(FOUND_SWIG)$(FOUND_PERL5),11)
-include perl5/Makefile.snippet
-endif
-
-ifeq ($(FOUND_SWIG)$(FOUND_PYTHON),11)
-include python/Makefile.snippet
-endif
-
-ifeq ($(FOUND_SWIG)$(FOUND_RUBY),11)
-include ruby/Makefile.snippet
-endif
-
 .PHONY: all clean install install_spl
 
 include Makefile.deps
